lib/process-services/src/lib/task-list/components/task-filters.component.spec.ts

```diff
@@ -351,4 +363,30 @@ describe('TaskFiltersComponent', () => {
         await fixture.whenStable();
         expect(component.currentFilter).toBe(undefined);
     });
+
+    describe('Display Task Filters', () => {
+
+        it('Should be able to display default task filters', async  () => {
+            spyOn(taskFilterService, 'getTaskListFilters').and.returnValue(of([]));
+            const defaultTaskFiltersMock = [
```

Comment:
Shouldn't we check also the `Queued Task` filter as being one of the default task filters?